docker/prod.yml

```diff
@@ -6,12 +6,16 @@ services:
     restart: always
   backend:
     restart: always
+    networks: 
+    - webnet
   incron:
     restart: always
   minion:
     restart: always
   frontend:
     restart: always
+    networks:
+    - webnet
```

Comment:
I think you should either:
- put `webnet` to every services
- or put `webnet` and `default` to services you want to monitor

In you config, postgres  / incron stays on default, while backend / frontend would only be on webnet

(or Am I wrong ?)

Reply:
What does `default` add ? The idea is just to have `frontend`, `backend` and the `apache_exporter`, `nginx_exporter` on the same network so that they can communicate.

Reply:
default is the default network, where all service without a networks directive are located :-)

Reply:
if you want backend to be able to communicate with postgres, it will have to be in default network (as well as webnet).